lgsf/scrapers/base.py

```diff
@@ -5,9 +5,10 @@
 import traceback
 
 import requests
-import requests_cache
 
-requests_cache.install_cache("scraper_cache", expire_after=60 * 60 * 24)
+# import requests_cache
+
+# requests_cache.install_cache("scraper_cache", expire_after=60 * 60 * 24)
```

Comment:
To explain why this is hear: When working on a single scraper there's a load of iterating, especially in HTML parsing scrapers. This speeds up the iterations by caching the whole request for a short time. I wrote a load of these on trains, so was especially useful.

I think the ideal here would be to use a flag that enables it. We might want to consider a "develop mode" flag even, one that enables this cache, disables `rich`, turns on verbose logging, etc etc.

Either way, fine to remove for now, it's just a progressive enhancement for developers

Reply:
Sounds liek a good idea - #16